deploy/olm-catalog/portworx/portworx.package.yaml

```diff
@@ -1,7 +1,7 @@
 packageName: portworx-certified
 channels:
 - name: alpha
-  currentCSV: portworx-operator.v23.4.0
+  currentCSV: portworx-operator.v23.4.1
 - name: stable
-  currentCSV: portworx-operator.v23.4.0
+  currentCSV: portworx-operator.v23.4.1
```

Comment:
I think we had some trouble with this file in the lates builds -- we'll need to check via Jenkins builds